packages/typography/src/Typography.tsx

```diff
@@ -136,4 +138,166 @@ const Overline: ExtendableBox<{
   return <Box className={cx(sharedStyles, overline, className)} {...rest} />;
 };
 
-export { H1, H2, Subtitle, Body, InlineCode, Disclaimer, Overline };
+const OpenInNewTab = (props: JSX.IntrinsicElements['svg']) => {
+  return (
+    <svg
+      width="16px"
+      height="16px"
+      viewBox="0 0 16 16"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      {...props}
+    >
+      <title>Open in New Tab</title>
+      <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+        <path
+          d="M9.29411765,3 L10.7727647,4.47864706 L6.42788235,8.82352941 L7.17647059,9.57211765 L11.5218824,5.22776471 L13,6.70588235 L13,3 L9.29411765,3 Z M4,4.58823529 L4,12 L11.4122941,12 L11.4122941,7.76470588 L10.3529412,7.76470588 L10.3529412,10.9411765 L5.05882353,10.9411765 L5.05882353,5.64705882 L8.23529412,5.64705882 L8.23529412,4.58823529 L4,4.58823529 Z"
+          fill="#007CAD"
+        />
+      </g>
+    </svg>
+  );
+};
+
+const anchorDataProp = createDataProp('anchor-container');
+
+const linkStyles = css`
+  text-decoration: none;
+  color: ${uiColors.blue.base};
+  cursor: pointer;
+
+  span {
+    position: relative;
+
+    &:after {
```

Comment:
This approach worked great for the MongoNav, but unfortunately, this will break sometimes when being used in traditional body copy if the link text wraps.

If you'd like to see the rabbit-hole this problem sent medium down, here's a good read for ya: https://medium.design/crafting-link-underlines-on-medium-7c03a9274f9

I think we'll need to get clever here...